services/src/main/java/org/keycloak/services/managers/ApplianceBootstrap.java

```diff
@@ -127,4 +139,25 @@ public void createMasterRealmUser(String username, String password) {
         ServicesLogger.LOGGER.addUserSuccess(username, Config.getAdminRealm());
     }
 
+    public void createTemporaryMasterRealmAdminService(String clientId, String clientSecret, Integer expriationMinutes) {
```

Comment:
I realized we actually don't test this anywhere, meaning we don't know if service acc created this way actually works as it's supposed to, i.e. it's able to actually log in.

Could you please add some test for it?

Reply:
Added the ability to invoke kcadm from the raw distribution and added a test using the service account.